packages/design-system/src/icons/svg.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+export { default as arrowCircle } from './arrow_circle.svg';
```

Comment:
Alternatively, maybe just update `icons/index.js` to do `export { ReactComponent as ArrowCircle, default as ArrowCircleRaw } from './arrow_circle.svg';`?

Feels a bit cleaner than this separation here.

Reply:
Then the Storybook stories for the icons would have to be amended to filter this value out, because it just imports `* as icons` from here. See [this conversation](https://github.com/google/web-stories-wp/pull/10159#discussion_r782541788).

But that's of course doable too. I went a bit back and forth on what to do here and have no strong opinions one way or the other.

Reply:
That shouldn't be too difficult to do I think.

cc @BrittanyIRL for thoughts

Reply:
I'm honestly fine with either approach personally. Updating storybook is doable, I don't really see the benefit from swapping approaches. Think what @barklund did here makes sense and is readable but @swissspidy if you feel strongly about it then I'd be cool with it.

Reply:
Don't feel strongly about it, but I do think we should at least update the name.

The original icon is called `ArrowCircle`, not `arrowCircle` as here.